app/cni/pkg/cni/main.go

```diff
@@ -85,13 +91,43 @@ func parseConfig(stdin []byte) (*PluginConf, error) {
 	return &conf, nil
 }
 
+func hijackMainProcessStderr() *os.File {
+	file, err := getCniProcessStderr()
+	if err != nil {
+		log.Error(err, "could not hijack main process file - continue logging to "+defaultLogLocation)
+		return nil
+	}
+	log.V(0).Info("successfully hijacked stderr of cni process - logs will be available in 'kubectl logs'")
+	os.Stderr = file
+	log = core.NewLoggerTo(os.Stderr, 2).WithName(defaultLogName)
+	return file
+}
+
+func getCniProcessStderr() (*os.File, error) {
+	pids, err := pidOf("/install-cni")
```

Comment:
maybe we could extract `/install-cni` to const at the top of the file, wdyt?

Reply:
that's the release branch, so let's move the discussion about it in different PR